src/uk/gov/hmcts/contino/GithubAPI.groovy

```diff
@@ -13,6 +13,8 @@ class GithubAPI {
     this.steps = steps
   }
 
+  private cachedLabelList = []
```

Comment:
static is gone?

Reply:
Based on my understanding, I don't believe that we want it static.  Making it so will decouple the cache from each instance of the github api class, which will break if multiple instances are created for different repos.